docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -135,25 +135,25 @@ The `active_swaps` method returns all the swaps that are currently running on th
 #### Command (without include_status)
 
 ```bash
-curl --url "http://127.0.0.1:7783" --data '
+curl --url "http://127.0.0.1:7783" --data "
```

Comment:
@smk762 Wouldn't a single-quoted body be more readable, if we use it everywhere? I think we already had some problems with CLI users due to forgotten/misplaced escape symbols.

Reply:
I agree - was aiming to make it more consistent, and told there might be some issues on windows with single quote - Will do some tests to confirm and change over if ok.

Reply:
Thanks :slightly_smiling_face:

Reply:
Yeah, looks like windows is spoiling the party here 

`curl.exe --url "http://127.0.0.1:7783/" --data "{"method":"version","userpass":"RPC_CONTROL_USERPASSWORD"}"`

`{"result":"2.1.4845_dev_72ff70357_Windows_NT_Release","datetime":"2021-11-01T20:12:18+07:00"}`


`curl.exe --url "http://127.0.0.1:7783/" --data '{"method":"version","userpass":"RPC_CONTROL_USERPASSWORD"}'`

`{"error":"expected value at line 1 column 1"}`

A few options:
- leave as is
- change to look cleaner without escapes, but have a note for windows users at the top of the sidebar submenu (like the note on Rational numbers)
- implement some kind of operating system detection and apply dynamic content based on this 

There is also the matter of whether windows user is using cmd or powershell, which does things differently. The above commands were in CMD. Powershell is a bit more complex.

@tonymorony @gcharang thoughts?

Reply:
> implement some kind of operating system detection and apply dynamic content based on this

we can do dynamic content based on OS detection. but, let's do dynamic content based on user selection. we can even store the preference as a cookie so it can be remembered on a subsequent visit.

for the interface, we can create a tab component and it will show the clean command (linux) by default. a user can click on the windows option if they want to see it. the escaping can be done on the fly on client side and we will only need to input the clean version in the docs. let me create a component for it. will ping you ( @smk762 ) once done

Reply:
also, let us support only one of powershell vs cmd on windows. whichever is preferred by devs on windows

Reply:
issue created here: https://github.com/KomodoPlatform/developer-docs/issues/304

Reply:
> also, let us support only one of powershell vs cmd on windows. whichever is preferred by devs on windows

In my opinion, CMD would be simplest to demonstrate an example, and look most like standard linux/mac commands. Powershell invokes some extra stuff which is a little inconvenient (see: https://newbedev.com/powershell-equivalent-of-curl) 
Any dev who prefers powershell would have no problem translating the CMD method into powershellese, though those preferring CMD might find translating back the other way a bit more difficult.

Reply:
I do disagree. Windows encourage to use windows new terminal which have a native PowerShell support - it's much better to use what's modern and available for the users.

Reply:
@smk762 @Milerius how about we recommend windows users to use git bash and link them to a tutorial like this: https://www.youtube.com/watch?v=-lIdKmf28xE ? I think linux commands of both types: `./komodo-cli getnewaddress` and `--url "http://127.0.0.1:7783" --data '{...............}'` will work without issue in the git bash terminal

Reply:
I'm in favour of nudging away from windows.

Reply:
> @smk762 @Milerius how about we recommend windows users to use git bash and link them to a tutorial like this: https://www.youtube.com/watch?v=-lIdKmf28xE ? I think linux commands of both types: `./komodo-cli getnewaddress` and `curl --url "http://127.0.0.1:7783" --data '{...............}'` will work without issue in the git bash terminal

I think it's not even required, most of windows user can now have official windows terminal, with native bash on linux and powershell support in it.